tests/test_mplutils.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from lsst.cst.mpl_utils import remove_figure
+
+
+class TestMPLUtils(unittest.TestCase):
+    """Test  matplotlib utils."""
+
+    def testRemoveFigure(self) -> None:
+        """Create a figure and test that the remove_figure function
+        removes it as expected."""
+
+        # Data for plotting
+        t = np.arange(0.0, 2.0, 0.01)
+        s = 1 + np.sin(2 * np.pi * t)
+
+        fig, ax = plt.subplots()
+        ax.plot(t, s)
+
+        ax.set(
+            xlabel="time (s)",
+            ylabel="voltage (mV)",
+            title="A simple test plot",
+        )
+        ax.grid()
+        self.assertIsNotNone(fig)
+
+        # Remove figure using utility function
+        remove_figure(fig)
```

Comment:
Is there any way to check that it's gone?

Reply:
Not that I could easily see. I'm not convinced the function is doing what it should .. but that is one of the improvements to be made to the contents of the package in future.

Reply:
I'm not saying this doesn't work, but I've never managed to make this work myself (though this does look a little different).

The way you could test if it's work is: make a loop that creates the figure more than 20 times, and call the remove inside the loop. If the "You have more than 20 open figures" warning appears, it's not working as you want, but if it doesn't, then that's a very good sign. You could do that in a test with the opposite of an `assertWarns` but that's obviously not very robust, as it will depend on matplotlib keeping that warning, and the warning limit staying at 20, but it's a thought/starting point.